package.json

```diff
@@ -30,6 +30,14 @@
   },
   "devDependencies": {
     "dotenv": "^8.2.0",
-    "nodemon": "^2.0.4"
+    "jest": "^26.4.2",
+    "nodemon": "^2.0.4",
+    "supertest": "^5.0.0"
+  },
+  "jest": {
+    "testEnvironment": "node",
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
+    ]
   }
```

Comment:
isn't this change already in the repository ?
![image](https://user-images.githubusercontent.com/14032427/95047246-5f4b5180-0703-11eb-8639-1e1f6b380240.png)

Reply:
I am little bit confused, should I just fork it again as a fresh one?